### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by veblush**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -11,74 +11,77 @@ for general contribution guidelines.
...
+- [sergiitk](https://github.com/sergiitk), Google LLC
 - [soheilhy](https://github.com/soheilhy), Google LLC
-- [sreecha](https://github.com/sreecha), LinkedIn
-- [srini100](https://github.com/srini100), Google LLC
 - [stanley-cheung](https://github.com/stanley-cheung), Google LLC
 - [veblush](https://github.com/veblush), Google LLC
 - [vishalpowar](https://github.com/vishalpowar), Google LLC
 - [Vizerai](https://github.com/Vizerai), Google LLC
-- [vjpai](https://github.com/vjpai), Google LLC
 - [wcevans](https://github.com/wcevans), Google LLC
```
Comment (ctiller) [inline MAINTAINERS.md:42]: I think wcevans can also be removed

File: MAINTAINERS.md [hunk 2]
```diff
@@ -11,74 +11,77 @@ for general contribution guidelines.
...
 - [dapengzhang0](https://github.com/dapengzhang0), Google LLC
 - [dfawley](https://github.com/dfawley), Google LLC
 - [dklempner](https://github.com/dklempner), Google LLC
+- [drfloob](https://github.com/drfloob), Google LLC
 - [ejona86](https://github.com/ejona86), Google LLC
-- [ericgribkoff](https://github.com/ericgribkoff), Google LLC
 - [gnossen](https://github.com/gnossen), Google LLC
 - [guantaol](https://github.com/guantaol), Google LLC
 - [hcaseyal](https://github.com/hcaseyal), Google LLC
 - [jboeuf](https://github.com/jboeuf), Google LLC
```
Comment (ctiller) [inline MAINTAINERS.md:25]: jbouef can be removed?

File: MAINTAINERS.md [hunk 3]
```diff
@@ -11,74 +11,77 @@ for general contribution guidelines.
 - [a11r](https://github.com/a11r), Google LLC
 - [apolcyn](https://github.com/apolcyn), Google LLC
 - [arjunroy](https://github.com/arjunroy), Google LLC
-- [AspirinSJL](https://github.com/AspirinSJL), Google LLC
 - [bogdandrutu](https://github.com/bogdandrutu), Google LLC
```
Comment (ctiller) [inline MAINTAINERS.md:14]: I think bogdan can be removed

**Review by ejona86 — APPROVED** [review]

File: MAINTAINERS.md (hunk 3 above)
Comment (veblush) [reply MAINTAINERS.md]: Done.

File: MAINTAINERS.md (hunk 2 above)
Comment (veblush) [reply MAINTAINERS.md]: Done.

File: MAINTAINERS.md (hunk 1 above)
Comment (veblush) [reply MAINTAINERS.md]: Done.

**Pull request merged**